compiler/luci/pass/src/QuantizationUtils.cpp

```diff
@@ -129,7 +129,6 @@ void compute_sym_scale_zp(float min, float max, float &scaling_factor, int64_t &
   if (scaling_factor < 1e-8)
     scaling_factor = 1e-8;
 
-  zp = 0;
```

Comment:
- Always `zp = 0` for sym
- `zp` is already allocated (_and initialized as zero_) by `compute_sym_scale_zp` caller.

Reply:
We may need to check caller must set zp to 0.

Reply:
I already checked existing code already initialized to zero since vector constructor with size initialize as zero.